### ISSUE TITLE
[Level 5] WIP

### ISSUE THREAD

**Issue by BetterToAutomateTheWorld** [MinetestForFun/server-minetestforfun-skyblock#134]
Ideas from the level 4 here (we don't use them all) https://github.com/MinetestForFun/server-minetestforfun-skyblock/issues/32
#### [Level 5] New nodes/features we want in quests
- At level 5, can use the PVP button to switch ON PVP with other PVP ON players (this is done already)
- *Teleport privilege at Level 6??? OR this:* https://github.com/MinetestForFun/server-minetestforfun-skyblock/issues/134#issuecomment-318850824
- Add aerbratus tree and cloud nodes
- Add *aerbratus* tools(better than diamond) (not sure what to call them yet)
- Enable Acacia trees
- Enable Aspen trees
- ...

#### [Level 5] quests steps

1) Goto /spawn to learn all about this server (rules, special items, commands, etc...)
2) Craft an Enchantment Table
3) Craft a sword
4) Enchant the sword with enchantment table.
5) Craft a shield
6) Mow the grass! Use Round-down to clear the weeds.
7) Harvest honey from a bee hive.
8) Place 2 Travelnet Boxes to set up a network
9) Craft a Work Bench
10) Dig 50 aerbratus leaves

#### [Level 5] quests rewards

1) Ender Chest
2) Trampoline
3) Acacia sapling
4) Aspen sapling
5) Round-down
6) Mailbox
7) 2 Travelnet Boxes
8) 2 Elevators
9) Aerbratus sapling
10) Aerbratus pick

(PS : *Italic* = not sure)

**Comment by aerozoic** [comment]
Are we going to use mods or should they only be vanillia?
:large_orange_diamond:

**Comment by ElementW** [comment]
@cessna151 The plans I had when I still contributed was to extend the `skyblock` mod with specific content and use that as quest tasks. Additions included a sky tree, special furnaces, day/night-specific nodes, etc...

**Comment by BetterToAutomateTheWorld** [comment]
We can use both. We need to finish the vanilla features/blocks and add some new features/blocks from mods.

**Comment by aerozoic** [comment]
With PVP soon to be granted at lvl 5, i think a few quests should be used to craft a sword and armor.

Also what do you think about giving teleport priv to lvl 6 players?
:large_orange_diamond:

**Comment by BetterToAutomateTheWorld** [comment]
I putted some of your ideas on the initial post, feel free to add more ideas by yourself ;)

**Comment by aerozoic** [comment]
Is it possible to detect when wearing a full set of armor? If so i think that would be good for quest 4. 
:large_orange_diamond:

**Comment by BetterToAutomateTheWorld** [comment]
Yes it's possible, we already did this on our classic server to activate some features when a full set of a class is wearing by the player.

**Comment by aerozoic** [comment]
Should we introduce aspen and/or acacia saplings as a reward?
:large_orange_diamond:

**Comment by aerozoic** [comment]
Will the Aerbratus code ever be finished? I doubt @ElementW will do it?
:large_orange_diamond:

**Comment by BetterToAutomateTheWorld** [comment]
I think the code is nearly done, but we need a confirmation from @ElementW

**Comment by ElementW** [comment]
@cessna151 @Darcidride The aerbratus code is incomplete and barely works. The sapling supposedly is functional and so are the trunk and leaves blocks, but the L-system defining the tree look is not working properly.

(edit: incomplete, not complete)

**Comment by aerozoic** [comment]
I thought it was incomplete because there are no crafting recipes and the sapling has a glitch where it's not replaced by a tree trunk when it grows. Also i thought the aerbratus was somehow related to the new cloud types.
:large_orange_diamond:

**Comment by aerozoic** [comment]
I've gotten several requests for this. Any idea when you want to get this finished?
:large_orange_diamond:

**Comment by BetterToAutomateTheWorld** [comment]
Just fill by updating the first post.
When we will have more than ten quest steps/rewards and a funny level with some new features we can code it after that

**Comment by aerozoic** [comment]
I've updated the list. We need suggestions about the final reward. I'll get some suggestions on IRC.
@ElementW Can you give us a basic explanation of how the aerbratus and new cloud types are supposed to work?
:large_orange_diamond:

**Comment by aerozoic** [comment]
<a href="#" title="**QUESTS** 1) Goto /spawn to learn all about this server (rules, special items, commands, etc...) 2) Craft an Enchantment Table 3) Craft a sword 4) Enchant the sword with enchantment table. 5) Craft a shield 6) Mow the grass! Use Round-down to clear the weeds. 7) Use the /mail command to send a message to another player. 8) Place 2 Travelnet Boxes to set up a network 9) Craft a Work Bench 10) Dig 50 aerbratus leaves **REWARDS** 1) Ender Chest 2) Trampoline 3) Acacia sapling 4) Aspen sapling 5) Round-down 6) Mailbox 7) 2 Travelnet Boxes 8) 2 Elevators 9) Aerbratus sapling 10) Aerbratus pick">I've updated the list.</a> After discussing on IRC, the plan for now is ElementW will fix the aerbratus tree growth and i will work on coding the craft recipes and new tools.

Does anybody know why acacia and aspen trees are not used? Is it ok to use them? @Darcidride 

I'm not sure teleport priv is a good idea for lvl 6.
:large_orange_diamond:

**Comment by aerozoic** [comment]
@ElementW, your turn :sunglasses: 
:large_orange_diamond:

**Comment by ElementW** [comment]
I'll get back to it Soon™, but I haven't yet come to any satisfying results using MT's tree generation facilities...

**Comment by aerozoic** [comment]
If we update to the latest version of [teleport request] then the "tp_tpc" priv would be a PERFECT reward for achieving LVL 6! https://github.com/ChaosWormz/teleport-request/blob/master/init.lua#L18
```
description = "Allow player to teleport to coordinates (if permitted by area protection)."
```
:large_orange_diamond:

**Comment by Lymkwi** [comment]
Small note : I cannot intercept chat commands, let alone those with complex arguments and a barely predictable behavior, without severely altering the mods which register them.

TL;DR I can't add chat commands feats without fùcking up the inbox and spawn mods. Find something else.

**Comment by aerozoic** [comment]
@LeMagnesium For quest 1 can't you just detect player location? I'll think of something else for quest 7.
:large_orange_diamond:

**Comment by aerozoic** [comment]
<a href="#" title="**QUESTS** 1) Goto /spawn to learn all about this server (rules, special items, commands, etc...) 2) Craft an Enchantment Table 3) Craft a sword 4) Enchant the sword with enchantment table. 5) Craft a shield 6) Mow the grass! Use Round-down to clear the weeds. 7) Harvest honey from a bee hive. 8) Place 2 Travelnet Boxes to set up a network 9) Craft a Work Bench 10) Dig 50 aerbratus leaves **REWARDS** 1) Ender Chest 2) Trampoline 3) Acacia sapling 4) Aspen sapling 5) Round-down 6) Mailbox 7) 2 Travelnet Boxes 8) 2 Elevators 9) Aerbratus sapling 10) Aerbratus pick">I've updated the list.</a> _(the href is just to preserve list edit for posterity.)_
:large_orange_diamond:
